src/payment/mod.rs

```diff
@@ -22,87 +22,3 @@ pub use store::{
 	ConfirmationStatus, LSPFeeLimits, PaymentDetails, PaymentDirection, PaymentKind, PaymentStatus,
 };
 pub use unified_qr::{QrPaymentResult, UnifiedQrPayment};
-
-/// Represents information used to send a payment.
-#[derive(Clone, Debug, PartialEq)]
-pub struct SendingParameters {
```

Comment:
First renaming this to `RouteParametersConfig` (the ldk-node one) in a separate commit may be helpful with review here. For next time.